### PR TITLE
Move bookmark message to correct new location

### PR DESCRIPTION
**Pull request by subsymbolic**
**Description**:
Move bookmark message to the tab view where the bookmark creation logic now lives

**Steps to test this PR**:
Create a bookmark, a confirmation message now appears

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR DRI Checklist:
- [x] Get advice or leverage existing code
- [x] Agree on technical approach with reviewer (if the changes are nuanced)
- [x] Ensure that there is a testing strategy (and documented non-automated tests)
- [x] Ensure there is a documented monitoring strategy (if necessary)
- [x] Consider systems implications (Database connections, Grafana stats, CPU)

### CODE REVIEW

File: DuckDuckGo/TabViewController.swift [hunk 1]
```diff
@@ -192,6 +196,12 @@ class TabViewController: WebViewController {
         navigationController?.isToolbarHidden = false
     }
     
+    private func makeToast(text: String) {
+        let x = view.bounds.size.width / 2.0
+        let y = view.bounds.size.height - 80
+        view.makeToast(text, duration: ToastManager.shared.duration, position: CGPoint(x: x, y: y))
```
Comment (nilnilnil) [inline DuckDuckGo/TabViewController.swift]: Hah. Has it always been called toast on iOS?

File: DuckDuckGo/TabViewController.swift [hunk 2]
```diff
@@ -192,6 +196,12 @@ class TabViewController: WebViewController {
         navigationController?.isToolbarHidden = false
     }
     
+    private func makeToast(text: String) {
+        let x = view.bounds.size.width / 2.0
+        let y = view.bounds.size.height - 80
```
Comment (nilnilnil) [inline DuckDuckGo/TabViewController.swift]: Are these Magic Numbers worth giving a const for?

Comment (subsymbolic) [inline DuckDuckGo/TabViewController.swift]: Good shout. I've left the 2.0 (it's just halving) and pulled out the 80.

File: DuckDuckGo/TabViewController.swift (hunk 1 above)
Comment (subsymbolic) [inline DuckDuckGo/TabViewController.swift]: It borrows from android :)

**Pull request merged**